apps/web/src/server/appeals/appeal-details/appeal-details.controller.js

```diff
@@ -19,6 +20,8 @@ export const viewAppealDetails = async (request, response) => {
 			? Object.values(appealDetails?.appealSite)?.join(', ')
 			: 'Address not known';
 
+		const startedAt = appealDetails?.startedAt && dateFormatter(appealDetails.startedAt);
+
 		const formattedAppeal = {
```

Comment:
I would probably move this to a mapper, where in the future we can add additional projections of the data for the upcoming screens.